packages/runtimes/js/src/bundle-url.js

```diff
@@ -37,6 +37,9 @@ function getBaseURL(url) {
 
 // TODO: Replace uses with `new URL(url).origin` when ie11 is no longer supported.
 function getOrigin(url) {
+  if (url.slice(0, 7) === 'file://') {
+    return 'file://';    
```

Comment:
The linter is complaining about the spaces after the semicolon

Reply:
Thanks. Sorry I didn't catch that. My auto-formatting was causing me headaches so I had it turned off.